kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -128,6 +128,15 @@ def uninstall(
                 f"Release with name {release_name} not found. Could not uninstall app."
             )
 
+    def template(self, release_name: str, chart: str, api_versions: str = "") -> str:
+        """
+        Render chart templates locally and display the output.
+        """
+        command = ["helm", "template", release_name, chart]
+        if not api_versions == "":
```

Comment:
From [PEP 8](http://www.python.org/dev/peps/pep-0008/), in the [“Programming Recommendations” section](https://www.python.org/dev/peps/pep-0008/#programming-recommendations):

> For sequences, (strings, lists, tuples), use the fact that empty sequences are false.

```suggestion
        if api_versions:
```

Reply:
done